internal/registry.go

```diff
@@ -23,6 +24,7 @@ var supportedServices = map[string]struct{}{
 	"droplets":   {},
 	"accounts":   {},
 	"spaces":     {},
+	"doks":       {},
```

Comment:
I'd rename this to be more user-friendly. 


```suggestion
	"kubernetes":       {},
```

Reply:
I think that this would be more confusing tbh. We are not offering any generic k8s tools, just tools for the doks apis..